<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Sponsors</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1031</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 238);</string>
    </property>
    <property name="title">
     <string>Sponsor</string>
    </property>
    <widget class="QTableView" name="tab_sponsor">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>150</y>
       <width>501</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_ID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Nom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom_société :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Adr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Adresse Mail :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>39</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: rgb(255, 255, 207);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>79</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 207);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhLowercaseOnly</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Adr">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>119</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">

border-color: rgb(0, 0, 255);
background-color: rgb(255, 255, 207);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhEmailCharactersOnly</set>
      </property>
      <property name="placeholderText">
       <string>exemple@gmail.com</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(214, 196, 0);
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Tarif">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tarif : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tarif">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 207);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Adr_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>N° Téléphone : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tlf">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>159</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 207);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Tarif_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Site Web : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_web">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 207);</string>
      </property>
      <property name="placeholderText">
       <string>www.exemple.com</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>300</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(214, 196, 0);
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_sup">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>470</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 207);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supp">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>460</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(216, 191, 0);</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>470</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>351</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Logo/Logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>tab_sponsor</zorder>
    <zorder>groupBox</zorder>
    <zorder>lineEdit_id_sup</zorder>
    <zorder>pb_supp</zorder>
    <zorder>label</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
